### DISCUSSION TITLE
Problem when sync sqlitedb with winform app deployment.

### DISCUSSION THREAD

**Discussion by AndyVu2311** [Mimetis/Dotmim.Sync#819] [Q&A] [answered]
I'm working on winform project using local sqlitedb and sync with Google Cloud Sql and everything works perfectly in debug. When testing on deployment using ClickOne, after first sync complete ( Dotmim able to connect to cloud sql and create _tracking tables and store procedures. but then Dotmim could not sync 2 databases, without any exception. When adding new rows to sqlite database the 
sync result return: Synchronization done but without any changes. 

Did anyone have this problem before? And what are your advice cause without any exception I don't really know what to do next.

**Comment by Mimetis** [comment]
Are you using a web api to expose the sync through a controller, and then using a `WebRemoteOrchestrator`, from the client side, or are you using a direction connection, using a `RemoteOrchestrator` ?

If you are behind a Web Api, maybe you don't see the error raised from the server side.
That's being said, you should have an error message on the client side ...

It's strange, I need more info to help your figure out what's happening here

**Reply by AndyVu2311** [reply]
I'm using direction connection with ssl.  Here are example code for sync action.
![image](https://user-images.githubusercontent.com/110625597/192017689-ffbe4fb7-5ebb-4351-9725-56a04a09fb83.png)

The thing is I can sync prefectly in debug and got this problem without any exception in deployment and I wonder I could be problem with .db file changes to DEPLOY file in application file.

**Reply by Mimetis** [reply]
in Debug mode, are you also syncing directly to google db or are u syncing with a local debug mysql server ?

**Reply by AndyVu2311** [reply]
in Debug I sync to Google directly using same function as Deploy.

**Reply by AndyVu2311** [reply]
More update information. I was testing with local mysql and the result is the same. Work perfectly fine in debug but in deployment, the behavior is the same - success connect  2 databases and make first sync but can upload changes from local sqlitedb to mysql so it's not problem with server. I'm gonna check local database and packages.config hopefully get any idea for solutions.

**Reply by Mimetis** [reply]
maybe you try with a new project, using a simple console application, and trying to make a simple sync with maybe 1 or 2 tables, and see if you've got the same result in debug / release

**Reply by AndyVu2311** [reply]
After checking all the file, In the first sync Dotmim create a .db file in the AppData/Local/Apps/2.0/Data difference location with the .db file the app read/write AppData/Local/Apps/2.0/..../exefile. Basically the app read/write in difference location with Dotmim, even though Dotmim and read/write function using same connection string (Data Source=.\LocalDB.db). And I'm still trying to find solution for that.

**Comment by AndyVu2311** [comment, marked as answer]
Found the solution. Default Dotmim read/write in difference location with the app when using same relative path (Data Source=.\Local.db). Change the connection string of app to AppData path.(Note that in AppData, the Local.db remain when uninstall app)
